mteb/evaluation/MTEB.py

```diff
@@ -168,6 +168,12 @@ def _display_tasks(self, task_list, name=None):
                     console.print(f"{prefix}{name}{category}{multilingual}")
                 console.print("\n")
 
+    def mteb_benchmarks(self):
+        """Get all benchmarks available in the MTEB."""
+        for benchmark in self._tasks:
+            name = benchmark.name
+            self._display_tasks(benchmark.tasks, name=name)
```

Comment:
Hmm on a second look. We might want to move the "display" code away from the MTEB object

(optional though)

Reply:
Hmm sure! Let's discuss more on that on a separate issue?